src/reducers/index.js

```diff
@@ -218,6 +239,21 @@ function initializeRowHeightsAndOffsets(state) {
   });
 }
 
+/**
+ * Initialize column offsets based on the given column props.
+ *
+ * @param {!Object} state
+ * @private
+ */
+function initializeColumnOffsets(state) {
+  const { scrollableColumns } = columnWidths(state);
+  const columnOffsetIntervalTree = new PrefixIntervalTree(scrollableColumns.map(column => column.width));
```

Comment:
construct the initial PrefixIntervalTree by storing each column's width

for now we have all the widths already calculated through our `columnWidths` selector